drivers/platform/xilinx/uart.c

```diff
@@ -149,17 +149,19 @@ int32_t uart_read(struct uart_desc *desc, uint8_t *data, uint32_t bytes_number)
 int32_t uart_write(struct uart_desc *desc, const uint8_t *data,
 		   uint32_t bytes_number)
 {
+#define WRITE_SIZE 255
```

Comment:
Shouldn't this be at the beginning of the file?

Reply:
Moved.